cli/create/templates/cartridge/init.lua

```diff
@@ -2,29 +2,14 @@
 
 require('strict').on()
 
-if package.setsearchroot ~= nil then
-	package.setsearchroot()
-else
-    -- Workaround for rocks loading in tarantool 1.10
-    -- It can be removed in tarantool > 2.2
-    -- By default, when you do require('mymodule'), tarantool looks into
-    -- the current working directory and whatever is specified in
-    -- package.path and package.cpath. If you run your app while in the
-    -- root directory of that app, everything goes fine, but if you try to
-    -- start your app with "tarantool myapp/init.lua", it will fail to load
-    -- its modules, and modules from myapp/.rocks.
-    local fio = require('fio')
-    local app_dir = fio.abspath(fio.dirname(arg[0]))
-    print('App dir set to ' .. app_dir)
-    package.path = app_dir .. '/?.lua;' .. package.path
-    package.path = app_dir .. '/?/init.lua;' .. package.path
-    package.path = app_dir .. '/.rocks/share/tarantool/?.lua;' .. package.path
-    package.path = app_dir .. '/.rocks/share/tarantool/?/init.lua;' .. package.path
-    package.cpath = app_dir .. '/?.so;' .. package.cpath
-    package.cpath = app_dir .. '/?.dylib;' .. package.cpath
-    package.cpath = app_dir .. '/.rocks/lib/tarantool/?.so;' .. package.cpath
-    package.cpath = app_dir .. '/.rocks/lib/tarantool/?.dylib;' .. package.cpath
-end
+-- configure path
+
+local path = require('app.path')
+local config = path.cfg()
+
+package.path = config.path
+package.cpath = config.cpath
+package.setsearchroot(config.root)
```

Comment:
Why do you do it here? We've moved this code to make `init.lua` more clear and simple. I expect smth like.
 
```lua
-- configure setsearchroot
require('app.path').cfg()
```